modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,79 @@ Module included in the following assemblies:
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on {product-title} 4.12 and later versions.
+
+This release ends maintenance support for OpenShift {SMProductShortName} version 2.2. If you are using OpenShift {SMProductShortName} version 2.2, you should upgrate to a supported version.
+
+[IMPORTANT]
+====
+{product-title} is designed for FIPS. When running on RHEL or RHEL CoreOS booted in FIPS mode, OpenShift Container Platform core components use the RHEL cryptographic libraries submitted to NIST for FIPS Validation on only the x86_64, ppc64le, and s390X architectures. For more information about the NIST validation program, see link:https://csrc.nist.gov/Projects/cryptographic-module-validation-program/validated-modules[Cryptographic Module Validation Program]. For the latest NIST status for the individual versions of the RHEL cryptographic libraries submitted for validation, see link:https://access.redhat.com/articles/2918071#fips-140-2-and-fips-140-3-2[Compliance Activities and Government Standards].”
+====
+
+=== Component versions included in {SMProductName} version 2.5
+
+|===
+|Component |Version
+
+|Istio
+|1.18
+
+|Envoy Proxy
+|1.24.12
+
+|Jaeger
+|1.47.0
+
+|Kiali
+|1.73.11
+|===
+
+=== OpenShift Service Mesh Console plugin
+
+This release introduces a generally available version of the OpenShift {SMProductShortName} Console (OSSMC) plugin. 
+
+The OSSMC plugin is an extension to the OpenShift Console that provides visibility into your Service Mesh. With the OSSMC plugin installed, a new Service Mesh menu option on the left-hand navigation of the web console, as well as new Service Mesh tabs that enhance existing Workloads and Service console pages. 
+
+The features of the OSSMC plugin are very similar to those of the standalone Kiali Console. The OSSMC plugin does not replace the Kiali Console, and after installing the OSSMC plugin, you can still access the standalone Kiali Console. 
+
+=== Cluster-Wide mesh migration
+
+This release adds documentation for migrating from a multitenant mesh to a cluster-wide mesh. For more information, see the following documentation:
+
+* About migrating to a cluster-wide mesh
+* Excluding namespaces from a cluster-wide mesh
+* Defining which namespaces receive sidecar injection in a cluster-wide mesh
+* Excluding individual pods from a cluster-wide mesh
+
+=== Gateway API CRD versions
+
+A new version of the Gateway API CRD is now available. Refer to the following table to determine which Gateway API version should be installed with the OpenShift {SMProductShortName} version you are using.
+
+|===
+|Service Mesh Version | Istio Version | Gateway API Version | Notes
+
+|2.5.x
+|1.18.x
+|0.6.2
+|Use experimental branch because `ReferenceGrand` is missing in v0.6.2
+
+|2.4.x
+|1.16.x
+|0.5.1
+|For multitenant mesh deployment, all Gateway API CRDs must be present. Use experimental branch.
+|===
+
```

Comment:
@yxun Please review the ARM Support entry and provide feedback. The entry was created using information from when ARM Support was introduced as Technology Preview and updated for GA. Thanks!

Reply:
Addressed in slack